### PR TITLE
Implement dark mode on homepage and tutorial search page

### PR DESCRIPTION
**Pull request by oyenuga17**
@SaySayo  please review.

### CODE REVIEW

**Comment by oyenuga17** [comment]
@sabine i really don't  know where tutorial_search.eml appears on the website so I only  changed the legacy colours I am familiar with, maybe you can please point me to it. Thank you.

**Comment by sabine** [comment]
It is here:

https://ocaml.org/docs/search?q=function

People arrive there from the documentation search bar.

**Comment by oyenuga17** [comment]
Screenshot 2024-02-20 at 11 37 10

I don't think this tertiary colour looks good on an orange text for the dark mode @sabine please what do you suggest.

**Comment by sabine** [comment]
Try this highlight color for the search terms: https://www.figma.com/file/eh8gAxaxrZT6RG3psHCm6M/Dark-%26-Light-mode-pages?type=design&node-id=373%3A26364&mode=design&t=GAweNYDVoW8NMd16-1

image

File: src/ocamlorg_frontend/pages/home.eml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 let package_card ~href ~img_path ~name description =
   <a href="<%s href %>" target="_blank"
-    class="bg-legacy-default dark:bg-legacy-dark-default text-legacy-default dark:text-legacy-dark-default p-4 rounded-xl flex md:gap-2 items-center legacy-card-hover">
+    class="bg-legacy-default dark:bg-legacy-dark-default text-title dark:text-dark-title  p-4 rounded-xl flex md:gap-2 items-center legacy-card-hover">
```
Comment (SaySayo) [inline src/ocamlorg_frontend/pages/home.eml]:
```suggestion
    class="bg-white dark:bg-dark-background text-title dark:text-dark-title  p-4 rounded-xl flex md:gap-2 items-center legacy-card-hover">
```

File: src/ocamlorg_frontend/pages/home.eml [hunk 2]
```diff
@@ -327,88 +325,80 @@ Layout.render
...
-          <h3 class="font-semibold">For Industrial Users</h3>
-          <div>
-            <%s! Icons.for_industrial_users "h-14 w-14" %>
-            <div class="absolute w-20 h-20 top-0 -right-10 rounded-full bg-legacy-primary-100"></div>
-          </div>
+          <h3 class="font-semibold text-title dark:text-dark-title">For Industrial Users</h3>
         </div>
         <div>
-          <div class="text-base text-legacy-default">
+          <p class="text-base text-content dark:text-dark-content">
```
Comment (SaySayo) [inline src/ocamlorg_frontend/pages/home.eml:355]:
```suggestion
          <p class="text-base text-title dark:text-dark-title">
```

File: src/ocamlorg_frontend/pages/home.eml [hunk 3]
```diff
@@ -417,13 +407,13 @@ Layout.render
...
-<div class="bg-legacy-default dark:bg-legacy-dark-default py-20 overflow-hidden">
+<div class="bg-white dark:bg-dark-background py-20 overflow-hidden">
   <div class="container-fluid">
     <div class="grid grid-cols-1 lg:grid-cols-2 items-center">
       <div class="space-y-8">
         <div class="space-y-6">
-          <h2 class="font-bold">OCaml Packages</h2>
-          <p class="text-lg text-legacy-lighter">
+          <h2 class="font-bold text-title dark:text-dark-title">OCaml Packages</h2>
+          <p class="text-lg text-title dark:text-dark-title">
```
Comment (SaySayo) [inline src/ocamlorg_frontend/pages/home.eml]:
```suggestion
          <p class="text-lg text-content dark:text-dark-content">
```

**Comment by oyenuga17** [comment]
@sabine I am now ready for a review, thank you.

**Review by sabine — APPROVED** [review]
LGTM

**Pull request merged**